canonical-websites/www.ubuntu.com
# Kubernetes docs scaffolding

## Done
Build the basic documentation theme in www.ubuntu.com

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to /kubernetes/docs/ and see the homepage of the k8s docs
- Go to /kubernetes/docs/logging and see an example of a default docs page

## Issue / Card
Fixes #4342
